lib/puppet/pops/adapters.rb

```diff
@@ -91,6 +91,7 @@ def extract_tree_text
       last = first + @adapted.length
       @adapted.eAllContents.each do |m|
         m_offset = m.offset
+        next if m_offset.nil?
```

Comment:
Unless this check is present, we should have a unit-test that verifies that all conceivable expressions gets the offset and length set. Do we have a test somewhere that tests all types of expressions for some other reason?